build/common.project.props

```diff
@@ -86,6 +86,11 @@
     <LangVersion Condition="'$(DotNetBuildFromSource)' == 'true'">latest</LangVersion>
   </PropertyGroup>
 
+  <!-- NuGet's dev branch always targets the next upcoming version of .NET, so we always use the preview SDK -->
+  <PropertyGroup Condition=" '$(IsEscrow)' != 'true' ">
```

Comment:
What does IsEscrow mean?

Reply:
Our VS insertion pipeline uses it to figure out if it needs to insert into the VS main branch, or a release branch. Therefore it's also a way to figure out if a specific commit is likely in our dev branch or a release branch.